dist/js/templateinside.js

```diff
@@ -219,6 +219,70 @@ baltimoreCounty.utility.formValidator = (function($) {
             requiredFieldPatternValidator: requiredFieldPatternValidator
         };
 })(jQuery);
+namespacer('baltimoreCounty.utility');
+
+baltimoreCounty.utility.format = (function () {
+    function formatCurrency(input) {
```

Comment:
Missing a `"use strict";` here

Reply:
@ascire  this wasn't supposed to be in this branch, but i have updated because this is valid...please review again.